lib/vector-core/src/event/lua/event.rs

```diff
@@ -9,6 +9,7 @@ impl<'a> ToLua<'a> for Event {
         match self {
             Event::Log(log) => table.raw_set("log", log.to_lua(lua)?)?,
             Event::Metric(metric) => table.raw_set("metric", metric.to_lua(lua)?)?,
+            Event::Trace(trace) => table.raw_set("trace", trace.to_lua(lua)?)?,
```

Comment:
Similar to VRL, I'm not sure that we need to support processing traces via lua right off the bat. Might be simpler to defer this until we have a use case and think a bit more about the API design.

Reply:
I think letting LUA on the side is probably a good idea, as opposed to VRL, I'm not sure this should be a priority. We could probably wait for user demand.